requirements/base-requirements.in

```diff
@@ -96,6 +98,7 @@ requests
 schema
 sentry-sdk
 sh
+shapely
```

Comment:
Same as above, can you please add a comment next to the dependency with the division owning this dependency and what feature(s) it's tied to? Thanks!